tests/TestProjectUsingNugetPackage/TestProjectUsingNugetPackage.fsproj

```diff
@@ -74,5 +69,5 @@
     <FSharpTargetsPath>$(MSBuildExtensionsPath32)\..\Microsoft SDKs\F#\3.1\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>
   </PropertyGroup>
   <Import Project="$(FSharpTargetsPath)" Condition="Exists('$(FSharpTargetsPath)')" />
-  <Import Project="..\packages\FsLexYacc.6.0.0\bin\FsLexYacc.targets" />
```

Comment:
when I ran the build locally everything passed, but I'm not totally sure. There are some more config adjustments I want to make so that the tests that need the nuget package use a locally built version, not the one from nuget